test/specs/commonTests.js

```diff
@@ -415,32 +415,30 @@ const _classNamePropValueBeforePropName = (Component, propKey, requiredProps) =>
 }
 
 /**
- * Assert that a Component correctly implements the "aligned" prop.
+ * Assert that a Component correctly implements columns prop.
  * @param {React.Component|Function} Component The component to test.
+ * @param {boolean} canEqual Flag that indicates possibility of "equal" value.
  * @param {Object} [requiredProps={}] Props required to render the component.
  */
-export const implementsAlignedProp = (Component, requiredProps = {}) => {
-  describe('aligned (common)', () => {
-    _definesPropOptions(Component, 'aligned')
-    _noDefaultClassNameFromProp(Component, 'aligned')
-    _noClassNameFromBoolProps(Component, 'aligned', requiredProps)
+export const implementsColumnsProp = (Component, canEqual, requiredProps = {}) => {
+  describe('columns (common)', () => {
+    _definesPropOptions(Component, 'columns')
+    _noDefaultClassNameFromProp(Component, 'columns', requiredProps)
+    _noClassNameFromBoolProps(Component, 'columns', requiredProps)
 
-    _.each(Component._meta.props.aligned, (propVal) => {
-      if (propVal === 'justified') {
-        it('adds "justified" without "aligned" to className', () => {
-          shallow(<Component { ...requiredProps } aligned='justified' />)
-            .should.have.className('justified')
+    it('adds prop value to className', () => {
```

Comment:
It looks like this test now uses `numberToWord` to convert the value.  Let's update the assertion to
```diff
-    it('adds prop value to className',
+    it('adds numberToWord value to className'
```